pkg/scheduler/testing/wrappers.go

```diff
@@ -182,7 +182,18 @@ func (c *ContainerWrapper) Resources(resMap map[v1.ResourceName]string) *Contain
 	}
 	c.Container.Resources = v1.ResourceRequirements{
 		Requests: res,
-		Limits:   res,
```

Comment:
`Resources` is directly/indirectly invoked by lots of places, let's keep it the same. Add another method `ResourceRequests` instead, similar to `ResourceLimits`.